src/lib/content.d.ts

```diff
@@ -356,6 +364,7 @@ type CAPIElement =
     | CaptionBlockElement
     | CalloutBlockElement
     | ChartAtomBlockElement
+    | QuizAtomBlockElement
```

Comment:
This is actually the only place where I find ordering things alphabetically useful (and I _know_ this will make @shtukas happy) so can we move this line and the declaration above to be better ordered. Hopefully the remaining code is also inline 🤞